src/components/SystemsTable/ConnectionStatusCol.js

```diff
@@ -39,11 +39,9 @@ const ConnectionStatusColumn = ({ connection_status, executor_type }) => {
           </Flex>
         }
       >
-        <Flex>
+        <Flex spaceItems={{ default: 'spaceItemsXs' }}>
```

Comment:
Theres duplicated work here, but eventually the goal is to delete the file completely once SystemsContent replaces the current implementation